net-im/qtox/qtox-1.7.0-r1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
Please try to use `repoman commit` for committing changes, which not only checks for QA problems but also fixes some such as the outdated copyright year.